lib/llm/providers/ollama.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module LLM
+  ##
+  # The Ollama class implements a provider for
+  # [Ollama](https://ollama.ai/)
+  class Ollama < Provider
+    require_relative "ollama/error_handler"
+    require_relative "ollama/response_parser"
+
+    HOST = "localhost"
+    DEFAULT_PARAMS = {model: "llama3.2", stream: false}.freeze
+
+    ##
+    # @param secret (see LLM::Provider#initialize)
+    def initialize(secret, **)
+      super(secret, HOST, 11434, ssl: false, **)
```

Comment:
I don't think this does what you expect. Keyword arguments won't "override" positional arguments. So, the syntax here `**` will have no effect on the host and port - right ? Let's not use syntax we don't understand, and revert back to something more simple:

```
    def initialize(secret, **options)
      super(secret, **{host: HOST}.merge(options))
   end
```